src/trap.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Generators
+ */
+
+const availableSignalsGenerator = (
+  suggestOptions?: Partial<Fig.Suggestion>
+): Fig.Generator => ({
+  script: "trap -l",
+  postProcess: (output) =>
+    output.split(" ").map((signal) => ({
```

Comment:
Also, I think this might be wrong - there can be a newline in the output. More reliable:
```suggestion
    output.split(/\s+/).map((signal) => ({
```

Reply:
No longer needed